src/testing_system/internal/adapter/assistants/API.py

```
@@ -0,0 +1,83 @@
+from langchain_core.messages import HumanMessage, SystemMessage, AIMessage
+
+
+class OpenAIAssistant(IAssistant):
+    def __init__(self, 
+                 model:str,
+                 base_url:str,
+                 temperature:Optional[float] = 0.9
+                 ):
+        try:
+            load_dotenv(dotenv_path="/workspace/chat-prep-etl/.env")
+            assert os.getenv("OPENAI_API_KEY") != None, "Need an API KEY to proceed with this adapter"
```

Ruff (0.15.12)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)